### ISSUE TITLE
On High Resolution Screen Putty Font Size Is Huge

### ISSUE THREAD

**Issue by GoogleCodeExporter** [jimradford/superputty#427]
```
What steps will reproduce the problem?
1. Use a laptop with a high res display (lenovo yoga 2 pro) 3200x1800
2. Open SuperPutty
3. Open an SSH session

What is the expected output? What do you see instead? The font size should be 
10pt, it is some randomly large font.  Simply going to the settings of Putty 
and just hitting apply fixes it.  You don't actually have to change anything, 
just simply hit apply.

What version of the product are you using? On what operating system? I am usig 
version 1.4.0.4.


Please provide any additional information below. I am running windows 8.1 
enterprise.

```

Original issue reported on code.google.com by `emel...@gmail.com` on 6 May 2014 at 2:02

**Comment by jimradford** [comment]
Closing this issue but all hope is not lost! We still want to help.
See: https://github.com/jimradford/superputty/wiki/Legacy-Imported-Issues on how to help us to help you!

**Comment by cbj4074** [comment]
Oh, *my*!

I just have to take a moment to thank `0xdabba...@gmail.com` for solving something has rubbed me wrong ever since I bought a 4K display: the horribly pixelated fonts in PuTTY!

You, sir, are a true hero. No matter which font I chose, nor at what size, fonts in PuTTY *always* looked abysmal! Even though it's only somewhat related, and I am using "stock" PuTTY (not SuperPutty), I stumbled upon your comment here and saw your suggestion to disable DPI awareness (also referred to as "display scaling" in later Windows versions).

And... ***BAM!!!***

The solution was simply to right-click on `putty.exe`, go to the `Compatibility` tab, and check `Disable display scaling on high DPI settings`:

![display-disable-scaling](https://cloud.githubusercontent.com/assets/1236883/11601524/c67775d8-9aa1-11e5-83f4-0ee566319741.png)

After making this change, the font is buttery-smooth, at 3840px wide. Madness!!!

I feel obligated to re-post this individual's sentiment from the old Google Code repository:

> OMFG - @Oxdabba...@gmail.com I could bloody kiss you. Literally like a massive mankiss, right on the chops. You, Sir, are the King of Men.

I couldn't agree more.
